prawns/tips/typegen/kusamaTypegen.json

```diff
@@ -0,0 +1,11 @@
+{
+  "outDir": "../src/types/kusama",
+  "chainVersions": "../../../metadata/kusamaVersions.json",
+  "typesBundle": "kusama",
+  "events": [
+    "tips.NewTip"
```

Comment:
Do we need to add the other events we are processing here?

Reply:
We don't because the other events (status events) we don't read the data inside the event - we just take a look at the `method` and the hash param.

Because of that we don't need to generate types and create typeGetters for these events in order for this to work. 
We could but it would be some more files to maintain when we don't actually need them for the functionality.